### PR TITLE
Explorer updates

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Add pages for new dbt Explorer features:
- Model performance
- Project recommendations

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).

Adding new pages (delete if not applicable):
- [x] Add page to `website/sidebars.js`
- [x] Provide a unique filename for the new page

### CODE REVIEW

File: website/docs/docs/collaborate/model-performance.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Model performance"
+sidebar_label: "Model performance"
+description: "Learn about ."
+---
+
+dbt Explorer provides metadata on dbt Cloud runs for in-depth model performance and quality analysis. This feature assists in reducing infrastructure costs and saving time for data teams by highlighting where to fine-tune projects and deployments &mdash; such as model refactoring or job configuration adjustments.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/model-performance.md:7]: You might consider starting with the value (2nd sentence).

File: website/docs/docs/collaborate/model-performance.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+<LoomVideo id='98f33b3b7a374df0b7c04747eae6ef44' />
+
+:::tip Public preview 
+
+The model performance feature is now available in dbt Explorer! Check it out!
```
Comment (runleonarun) [inline website/docs/docs/collaborate/model-performance.md:13]: The second sentence might not be easily understood when English is someone's second language. Definitely bring on the excitement. Maybe "Go explore!" (ha ha)

File: website/docs/docs/collaborate/model-performance.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+<LoomVideo id='98f33b3b7a374df0b7c04747eae6ef44' />
+
+:::tip Public preview 
+
+The model performance feature is now available in dbt Explorer! Check it out! 
+:::
+
+## The Performance overview page 
+
+The Performance overview page presents a comprehensive analysis across all project models, helping to pinpoint areas for performance enhancement. It displays the longest-running models, those most frequently executed, and the ones with the highest failure rates during runs/tests. Data can be segmented by environment and job type which can offer insights into:
```
Comment (runleonarun) [inline website/docs/docs/collaborate/model-performance.md]: I also wonder if this paragraph would benefit from bringing starting with the benefits. Maybe something like:
```suggestion
You can pinpoint areas for performance enhancement by using the Performance overview page. This page presents a comprehensive analysis across all project models and displays the longest-running models, those most frequently executed, and the ones with the highest failure rates during runs/tests. Data can be segmented by environment and job type which can offer insights into:
```

File: website/docs/docs/collaborate/model-performance.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-performance-overview-page.png" width="80%" title="Example of Performance overview page"/>
+
+You can view historical metadata for up to the past three months. Select the time horizon using the filter, which defaults to a two-week lookback.
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/ex-2-week-default.png" title="Example of dropdown"/>
+
+## The Model performance tab
+
+The Model performance tab allows for historical performance analysis, showing trends in execution times, counts, and failures. Daily execution data includes:
```
Comment (runleonarun) [inline website/docs/docs/collaborate/model-performance.md]: Same feedback here about bringing the value up front.

File: website/docs/docs/collaborate/project-recommendations.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+title: "Project recommendations"
+sidebar_label: "Project recommendations"
+description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
+---
+
+:::tip Public preview
```
Comment (runleonarun) [inline website/docs/docs/collaborate/project-recommendations.md:7]: Should these be a snippet to ensure consistency on different pages and in future cahnges?

File: website/docs/docs/collaborate/project-recommendations.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+:::
+ 
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+
+Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing. This overview page includes two top-level metrics measuring the test and documentation coverage of the models in your project. 
+
+- **Model test coverage** &mdash; The percent of models in your project (models not from a package or imported via dbt Mesh) with at least one dbt test configured on them.
+- **Model documentation coverage** &mdash; The percent of models in your project (models not from a package or imported via dbt Mesh) with a description.
+
+These recommendations provide insight into how you can build a more well documented, well tested, and well built project, leading to less confusion and more trust.
```
Comment (runleonarun) [inline website/docs/docs/collaborate/project-recommendations.md]: Ooh definitely move this sentence up so google searches find it easily!

File: website/docs/docs/collaborate/project-recommendations.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+| Documentation | Undocumented Models | Models without a model-level description | [GitHub](https://dbt-labs.github.io/dbt-project-evaluator/0.8/rules/documentation/#undocumented-models) |
+| Documentation | Undocumented Source | Sources (collections of source tables) without descriptions | [GitHub](https://dbt-labs.github.io/dbt-project-evaluator/0.8/rules/documentation/#undocumented-sources) |
+| Documentation | Undocumented Source Tables | Source tables without descriptions | [GitHub](https://dbt-labs.github.io/dbt-project-evaluator/0.8/rules/documentation/#undocumented-source-tables) |
+| Governance | Public Model Missing Contract | Models with public access that do not have a model contract to ensure the data types | [GitHub](https://dbt-labs.github.io/dbt-project-evaluator/0.8/rules/governance/#public-models-without-contracts) |
+
+
+Models, sources and exposures each also have a Recommendations tab on their resource details page, with the specific recommendations that correspond to that resource:
```
Comment (runleonarun) [inline website/docs/docs/collaborate/project-recommendations.md]: This feels a little lost after the table. Can it include a heading?

**Review by runleonarun — COMMENTED** [review]
@nghi-ly This is really great and my comments are definitely non-blocking.  Just a few thoughts on making your super solid content more engaging and discoverable via searches or quick glances.

File: website/docs/docs/collaborate/model-performance.md (hunk 4 above)
Comment (nghi-ly) [reply website/docs/docs/collaborate/model-performance.md]: sounds good!

File: website/docs/docs/collaborate/project-recommendations.md (hunk 6 above)
Comment (nghi-ly) [reply website/docs/docs/collaborate/project-recommendations.md]: great idea!

**Review by runleonarun — APPROVED** [review]
:shipit:

**Pull request merged**